packages/portal-gun/src/internal/debug.ts

```diff
@@ -0,0 +1,7 @@
+import debugC from 'debug';
+
+export const debug = debugC('inox-tools:portal-gun')
+
+export const getDebug = (segment?: string) => {
```

Comment:
What does it do?

Reply:
It's part of my template, it isn't being used here.